engine/jobmaster/dm/api_test.go

```diff
@@ -166,8 +183,9 @@ func TestQueryStatusAPI(t *testing.T) {
 	messageAgent.On("SendRequest", mock.Anything, "task5", mock.Anything, mock.Anything).Return(loadStatusResp, nil).Once()
 	jm.workerManager.UpdateWorkerStatus(runtime.NewWorkerStatus("task6", frameModel.WorkerDMSync, "worker6", runtime.WorkerOnline, 3))
 	messageAgent.On("SendRequest", mock.Anything, "task6", mock.Anything, mock.Anything).Return(syncStatusResp, nil).Once()
+	jm.workerManager.UpdateWorkerStatus(runtime.NewWorkerStatus("task7", frameModel.WorkerDMLoad, "worker7", runtime.WorkerFinished, 3))
```

Comment:
this is a test bug

Reply:
why do you think this is a bug? it may be a test case 
cc @GMHDBJD

Reply:
because I have seen the finished worker would not be get 
@lance6716 @GMHDBJD  https://github.com/pingcap/tiflow/blob/0ad56ca6594aeaaff0da24c73c39f0100925d6b6/engine/jobmaster/dm/api.go#L100-L104

Reply:
revert it https://github.com/pingcap/tiflow/pull/7507/commits/ab7f02a91ba0b3ee797dcf86b729d0d93af21e11